Apps/Sandcastle/gallery/CZML Rectangle.html

```diff
@@ -0,0 +1,122 @@
+  <meta name="description" content="CZML Rectangle">
+  <meta name="cesium-sandcastle-labels" content="CZML">
+  <title>Cesium Demo</title>
+  <script type="text/javascript" src="../Sandcastle-header.js"></script>
+  <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.9/require.js"></script>
+  <script type="text/javascript">
+  require.config({
+    baseUrl : '../../../Source',
+    waitSeconds : 60
+  });
+  </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+@import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+  "use strict";
+  //Sandcastle_Begin
+  var czml = [
+    {
+      "id" : "document",
+      "name" : "CZML Geometries: Rectangle",
+      "version" : "1.0"
+    }, {
+      "id" : "redRectangle",
+      "name" : "extruded red rectangle with black outline",
+      "rectangle" : {
+        "show" : true,
+        "coordinates" : { "wsenDegrees" : [-120, 40, -110, 50] },
+        "height" : 600000,
+        "extrudedHeight" : 0,
+        "fill" : true,
+        "material" : {
+          "solidColor" : {
+            "color": {
+              "rgba" : [255, 0, 0, 100]
+            }
+          }
+        },
+        "outline" : true,
+        "outlineColor" : {
+          "rgbaf" : [0, 0, 0, 1.0]
+        }
+      }
+    }, {
+      "id" : "stripeRectangle",
+      "name" : "rectangle with blue/green stripes",
+      "rectangle" : {
+        "show" : true,
+        "coordinates" : { "wsenDegrees" : [-105, 40, -95, 50] },
+        "height" : 0,
+        "fill" : true,
+        "material" : {
+          "stripe" : {
+            "orientation" : "VERTICAL",
+            "evenColor" : { "rgba" : [0, 255, 0, 255] },
+            "oddColor" : { "rgba" : [0, 0, 255, 255] },
+            "repeat" : 5
+          }
+        }
+      }
+    },
+    {
+      "id" : "yellowRectangle",
+      "name" : "extruded rectangle with yellow outline",
+      "rectangle" : {
+        "show" : true,
+        "coordinates" : { "wsenDegrees" : [-90, 40, -80, 50] },
+        "height" : 600000,
+        "extrudedHeight" : 0,
+        "fill" : false,
+        "outline" : true,
+        "outlineColor" : {
+          "rgba" : [255, 255, 0, 255]
+        },
+        "outlineWidth" : 2,
+        "rotation" : 0.5
+      }
+    },
+    {
+      "id" : "textureRectangle",
+      "name" : "rectangle with image, above surface",
+      "rectangle" : {
+        "show" : true,
+        "coordinates" : { "wsenDegrees" : [-75, 40, -50, 45] },
+        "height" : 600000,
+        "fill" : true,
+        "material" : {
+          "image" : {
+            "image" : { "uri" : "/Apps/Sandcastle/images/Cesium_Logo_Color.jpg" }
```

Comment:
You'll note the absolute path is not working on [tiffanylu.com](http://tiffanylu.com/cesium/Apps/Sandcastle/index.html?src=CZML%20Rectangle.html&label=CZML) anyway, because it's deployed to a subfolder.  The relative path should fix it there too.